File: PipelineService/Models/MqttMessages/SimpleBlockExecutionResponse.cs
using System;

namespace PipelineService.Models.MqttMessages
{
    public class SimpleBlockExecutionResponse : BlockExecutionResponse
    {

        /// <summary>
        /// The id of the resulting dataset stored in the Dataset Storage.
        /// </summary>
        public Guid? ResultDatasetId { get; set; }
    }
}

File: BlockWorker/src/models/simple_block_execution_response.py
from src.models.block_execution_response import BlockExecutionResponse


class SimpleBlockExecutionResponse(BlockExecutionResponse):
    result_dataset_id: str

    def __init__(self) -> None:
        super().__init__()

    def set_result_dataset_id(self, id: str) -> None:
        self.result_dataset_id = id

    def get_result_dataset_id(self) -> str:
        return self.result_dataset_id
